### ISSUE TITLE
Troubles with Ignore File

### ISSUE THREAD

**Issue by jherdman** [jeremyjh/dialyxir#287]
### Environment

* Elixir & Erlang/OTP versions (elixir --version): Elixir 1.8.0 (compiled with Erlang/OTP 21)

* Which version of Dialyxir are you using? (cat mix.lock | grep dialyxir):   `"dialyxir": {:hex, :dialyxir, "1.0.0-rc.4", "71b42f5ee1b7628f3e3a6565f4617dfb02d127a0499ab3e72750455e986df001", [:mix], [{:erlex, "~> 0.1", [hex: :erlex, optional: false]}]},`

### Current behavior

I have a file that references some third party code with a `:no_return` error. I'm trying to ignore this error until the upstream library can resolve the issue. I've added a `.dialyzer_ignore.exs` file that looks like this:

```elixir
[
  {"lib/my_app/feed_reader/feed_reader.ex"}
]
```

I've also tried setting a config option in `mix.exs`:

```elixir
ignore_warnings: "lib/my_app/feed_reader/feed_reader.ex"
```

I see this:

```
lib/my_app/feed_reader/feed_reader.ex:15:no_return
Function get_attrs_for_source/1 has no local return.
________________________________________________________________________________
lib/my_app/feed_reader/feed_reader.ex:34:no_return
Function get_attrs_for_entries/1 has no local return.
________________________________________________________________________________
lib/my_app/feed_reader/feed_reader.ex:47:no_return
Function get_feed/1 has no local return.
```

### Expected behavior

Dialyzer should helpfully ignore these errors, but sadly does not :(

**Comment by jherdman** [comment]
I've resolved my issue. I realized after a more careful reading of the documentation that `ignore_warnings` is meant to point to a file, not the file you want to ignore. That said, I kind of though the purpose of #272 was to make ".dialyzer_ignore.exs" recognized by default?

**Comment by asummers** [comment]
It should. What does your dialyzer config look like in your mix.exs?

**Comment by jherdman** [comment]
Right now it looks like this:

```elixir

  def project do
    [
      app: :my_app,
      version: "0.0.1",
      elixir: "~> 1.7",
      elixirc_paths: elixirc_paths(Mix.env()),
      compilers: [:phoenix, :gettext] ++ Mix.compilers(),
      start_permanent: Mix.env() == :prod,
      aliases: aliases(),
      deps: deps(),
      dialyzer: dialyzer()
    ]
  end

  defp dialyzer do
    [
      plt_add_deps: :transitive,
      ignore_warnings: ".dialyzer_ignore.exs"
    ]
  end
```

If I remove the `ignore_warnings` line I see the above issue.

**Comment by asummers** [comment]
Interesting. That should be working. Thanks for the report!

**Comment by asummers** [comment]
I know what's happening. We just need a release bump @jeremyjh

**Comment by jherdman** [comment]
You beat me to it. I just tried edge and it worked perfectly. I think we can safely close this issue.

**Comment by jeremyjh** [comment]
Released in `1.0.0-rc.5`
